vision_system_node:
  ros__parameters:
    color_image_topic: "/head_front_camera/rgb/image_raw"
    depth_image_topic: "/head_front_camera/depth_registered/image_raw"
    camera_info_topic: "/head_front_camera/depth_registered/camera_info"
    frames_approx_sync: false

    post_processing:
      package: "examples"
      module: "post_processing_example"
      class: "PostProcessingExample"

  
